src/main/java/twistedgate/immersiveposts/common/blocks/HorizontalTrussBlock.java

```diff
@@ -55,7 +56,7 @@ public class HorizontalTrussBlock extends GenericPostBlock implements IPostBlock
 	public static final EnumProperty<EnumHTrussType> TYPE = EnumProperty.create("type", EnumHTrussType.class);
 	
 	public HorizontalTrussBlock(IPostMaterial material){
-		super(material, "_truss");
+		super(material);
```

Comment:
Honestly at this point i've gotten too lazy now to even ask why this was removed.

(Having already written stuff in the finish-review)

Reply:
Pressed on close by mistake.

I'm sorry for touching the API, I was really looking forward cause I needed a 1.20.1 version of the mod working as soon as possible. I will redo everything from 0 tomorrow (it's 4 AM rn in Italy). Didn't want to offend you, just left too many comments probably ;) I'm really sorry, hope I can do it better tomorrow and we'll see

Reply:
My plan was to actualy skip all of the 1.20 versions from 0 to 3 and just go straight to 1.20.4 using *NeoForge* instead.

I have just been dead about it because i've been trying to "recharge", if that makes sense.

Reply:
Still, a 1.20.1 version should not be bad (for example, I'm using 1.20.1 for my server at the moment). If I fix everything, I would really be happy to see an official 1.20.1 version

Reply:
Honestly, im not gonna go against my plan.

If you want to port this to 1.20.4 im all for it, otherwise don't waste your time and i will do it myself as usual